### DISCUSSION TITLE
Avoid `Esc` clear chat text

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#19403] [Ideas] [answered]
For example, I type some text in chat panel, and I want to leave for get some beatmap information(like beatmap name), so I press esc to exit chat panel, but it clear the text or not exit(and it cannot undo) and painful
if I use `F8` or click chat panel button to exit the text I type will keep

And this approach is also in line with the current behavior of stable

**Comment by nagi-desuuu** [comment]
Duplicate of https://github.com/ppy/osu/issues/8715 ?

**Reply by cdwcgt** [reply]
I don't think so.
maybe I just want to check profile or something else, or not copy link

**Reply by cdwcgt** [reply]
And the core of the problem is `Esc` should not clear the textbox on this occasion
First, it does not conform to the behavior of stable.
Second, I think it's a counter-intuitive setting.

**Comment by peppy** [comment, marked as answer]
#19543
